Vocable/Common/Views/GazeableAlertController/GazeableAlertViewController.swift

```diff
@@ -293,7 +293,7 @@ final class GazeableAlertViewController: UIViewController, UIViewControllerTrans
         containerStackView.addArrangedSubview(actionButtonStackView)
 
         messageLabel.translatesAutoresizingMaskIntoConstraints = false
-        messageLabel.accessibilityIdentifier = "alert_message"
+        messageLabel.accessibilityIdentifier = AccessibilityID.shared.alert.messageLabel.id
```

Comment:
Because this is a UILabel element (as opposed to a config) I believe we can use the AccessibilityID property to simplify the identifier assignment, to something like:
```suggestion
        messageLabel.accessibilityID = .shared.alert.messageLabel
```